.gitmodules

```diff
@@ -22,3 +22,6 @@
 	path = Silicon/Arm/TFA
 	url = https://github.com/ARM-software/arm-trusted-firmware.git
 	branch = v2.7
+[submodule "Common/MU_FEATURE_DFCI"]
```

Comment:
Any thoughts on using git dependency for this repo?

Reply:
There is currently unsupported functionality if we pull it in using git dependencies. Close this conversation now.